testsuite/testsuite.c

```diff
@@ -210,7 +210,11 @@ int testsuite_test(int argc, char** argv)
 #endif /* HAVE_WNR */
 
     printf("\nAll tests passed!\n");
+#ifdef WOLFSSL_RIOT_OS
+    exit(0);
+#else
     return EXIT_SUCCESS;
+#endif
 }
```

Comment:
Perhaps we can create a single macro to handle exiting tests to keep this clean?

#ifdef WOLFSSL_RIOT_OS
#define EXIT_TEST(ret) exit(ret)
#else
#define EXIT_TEST(ret) return ret;
#endif

Recommend putting this macro logic somewhere common for these like wolfssl/wolfcrypt/types.h.